contracts/Account.sol

```diff
@@ -696,6 +696,38 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         return (pendingWithdrawal.value, pendingWithdrawal.timestamp);
     }
 
+    /**
+     * @notice Votes on a proposal in the referendum stage.
+     * @param proposalId The ID of the proposal to vote on.
+     * @param index The index of the proposal ID in `dequeued`.
+     * @param yesVotes The yes votes weight.
+     * @param noVotes The no votes weight.
+     * @param abstainVotes The abstain votes weight.
+     */
+    function voteProposal(
```

Comment:
nit : from the governance method we are invoking this seems to be a partial vote, do you think i'll make sense to reflect that same name here?

i.e voteProposal  -> votePartially